lib/seoshop-api/client.rb

```diff
@@ -159,5 +159,13 @@ def convert_hash_keys(value)
           value
       end
     end
+
+    def create_order(checkout_details, payment_details, shipping_details)
+      Seoshop::Client::Order.new(self, checkout_details, payment_details, shipping_details)
+    end
+
+    def create_ordered_product(checkout_id, shop_code, quantity, price)
+      Seoshop::Client::OrderedProduct.new(self, checkout_id, shop_code, quantity, price)
```

Comment:
would this not be easier or the same as 

like:
```
def create_order_product(checkout_id, details)
 response = post("#{@language}/checkouts/#{@checkout_id}/products.json", details)
 response.status == 201
end
create_order_product(191, { variant: @shop_code, quantity: @quantity, special_price_inclusive: @special_price })
```
which is exactly similar to other methods: https://github.com/Shop2market/seoshop-api/blob/master/lib/seoshop-api/api/shop_tracking.rb#L14